engine/lib/luajit-ffi-gen/src/enum_item/generate_lua.rs

```diff
@@ -1,19 +1,19 @@
 use super::EnumInfo;
 use crate::args::EnumAttrArgs;
-use crate::ffi_generator::FfiGenerator;
+use crate::ffi_generator::FFIGenerator;
```

Comment:
Didn't compiler show a warning about this? It should suggest `FfiGenerator` according to its rules.

Reply:
Hmm, I didn't check actually. I simply hated how the acronym looks :)

Reply:
Both `cargo build` and `cargo clippy` seem happy with this.